modules/channel-slack/src/backend/index.ts

```diff
@@ -46,7 +46,7 @@ const entryPoint: sdk.ModuleEntryPoint = {
   onBotUnmount,
   definition: {
     name: 'channel-slack',
-    menuIcon: 'none',
+    menuIcon: 'dot',
```

Comment:
You don't need to add icons to channels since they do not expose any interface! They only run on the backend.

Reply:
Ok, I was confused 😅  . I check the code and the icon menu is not required. https://github.com/botpress/botpress/blob/master/src/bp/sdk/botpress.d.ts#L215 I will remove this Key/Value when the `noInterface: true`

What do you think ?

Reply:
Sounds good. That's what I would do!